gen/build/src/lib.rs

```diff
@@ -129,6 +132,8 @@ fn build(rust_source_files: &mut dyn Iterator<Item = impl AsRef<Path>>) -> Resul
     build.cpp(true);
     build.cpp_link_stdlib(None); // linked via link-cplusplus crate
     build.include(&include_dir);
+    // Set feature parameters, such as C++ std, for simpler usage
+    build.flag_if_supported(cxxbridge_flags::STD);
```

Comment:
In the future, we can add other feature, such as folly or absl support. cargo-build clients only will have to chose the feature in their Cargo.toml.